airflow/models.py

```diff
@@ -2630,6 +2630,65 @@ def xcom_pull(
             include_prior_dates=include_prior_dates)
 
 
+class ExecutorQueue(Base):
+    __tablename__ = "executor_queue"
+
+    key = Column(String(ID_LEN), primary_key=True)
+    id = Column(String(ID_LEN), nullable=False)
```

Comment:
I think that ExecutorQueue, like LocalTaskJobs, should include the hostname.

Reply:
That doesn't make sense really, the queue is shared across all (potential) executors and across the celery cluster. Workers reporting back can happen at any of the executors and then the message is gone.